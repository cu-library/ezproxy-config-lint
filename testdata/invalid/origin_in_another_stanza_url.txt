Title Siku Quanshu
URL http://skqs.yourlib.org
DJ skqs.yourlib.org

Title Siku Quanshu Two
URL http://skqs2.yourlib.org
HJ http://skqs.yourlib.org
DJ skqs.yourlib.org
