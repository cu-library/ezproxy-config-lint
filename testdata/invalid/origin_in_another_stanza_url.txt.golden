testdata/invalid/origin_in_another_stanza_url.txt:7: HJ http://skqs.yourlib.org ← Origin already seen at "testdata/invalid/origin_in_another_stanza_url.txt:2" (L2002)
